Gopkg.lock

```diff
@@ -16,22 +16,6 @@
   pruneopts = "T"
   revision = "99064174e013895bbd9b025c31100bd1d9b590ca"
 
-[[projects]]
```

Comment:
please remove this file from the pr, since no actual packages were added or removed